storefront/app/javascript/spree/storefront/controllers/card_validation_controller.js

```diff
@@ -44,10 +44,10 @@ export default class extends Controller {
     if (validation.card) {
       // Update CVV validation length based on card type
       this.cvvTarget.setAttribute('maxlength', validation.card.code.size)
-      
+
       // Update card type field and icon
       const cardType = validation.card.type
-      this.ccTypeTarget.value = cardType
+      this.ccTypeTarget.value = cardType.replace(/-/g, '_')
```

Comment:
This is the only actual change, the rest was just formatting from a pre-commit hook.